internal/moduletest/mocking/overrides.go

```diff
@@ -96,52 +102,36 @@ func (overrides *Overrides) IsOverridden(module addrs.ModuleInstance) bool {
 	return false
 }
 
-// IsDeeplyOverridden returns true if an ancestor of this module is overridden
-// but not if the module is overridden directly.
+// GetResourceOverride checks the overrides for the given resource instance.
+// This function automatically checks if the containing resource has been
+// overridden if the instance is instanced.
```

Comment:
Not sure I understand that last sentence

Reply:
Clarified!